commands/audit/jas/common.go

```diff
@@ -24,7 +25,9 @@ import (
 )
 
 const (
-	NodeModulesPattern = "**/*node_modules*/**"
+	NodeModulesPattern          = "**/*node_modules*/**"
+	jfPackageManagerEnvVariable = "AM_PACKAGE_MANAGER"
+	jfLanguageEnvVariable       = "AM_LANGUAGE"
```

Comment:
maybe add it to analyzermanager.go instead? the other env var for it defifned there